client/src/partials/stock/dashboard/dashboard.html

```diff
@@ -141,8 +148,15 @@
             </tr>
           </thead>
           <tbody>
-            <tr>
-              <td colspan="4" ng-if="StockDashCtrl.donations.length === 0"><loading-indicator></loading-indicator></td>
+            <tr ng-if="StockDashCtrl.indicators.donations">
```

Comment:
Just curious, how an object called ```indicators``` can have a property ```donations```.
Simple variable should not be in plural, except array.